facades.md

```diff
@@ -0,0 +1,49 @@
+# Facade data in third-party-web
+
+In addition to identifying entities, third-party-web also includes data on entities' _products_, and available alternative libraries for products known as _facades_. A **third-party product** is a third-party resource that is included on a page, for example: a YouTube embed iframe. A **facade** is a frontend element which looks similar to the actual product and lazily loads it upon user interaction.  It's expected to be significantly more lightweight/faster. For example: [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed).
+
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) 7.0 has a new audit that identifies third-party 
+embeds (link coming soon) that can be lazy loaded with a facade. The Lighthouse audit is powered by the facade data in third-party-web.
+
+## Criteria for adding a new facade
+### Basic functionality
+
+* Loads an HTML component on page load which looks like the actual third-party 
+  embed.
+* Replaces that component with the actual third-party embed after some user 
+  interaction.
+    * User clicks on it, mouses over, etc.
+
+### Well Maintained
+
+* The projects issues and contributions are managed responsibly. Bugs are 
+  handled swiftly.
+* Is already used in production, ideally by users other than the creator.
```

Comment:
Would this requirement disqualify some of the options we were looking to recommend earlier? e.g Alex's vimeo lite component. I'm not saying this shouldn't be a recommendation (rather, I think production use is a great requirement), just pointing this out... :)

Reply:
I think it would disqualify many, but I think that's why they're guidelines w/case-by-case determination instead of strict requirements and the intention was just to not broadcast that.